src/main/java/org/sigmah/shared/computation/instruction/Pick.java

```diff
@@ -0,0 +1,56 @@
+package org.sigmah.shared.computation.instruction;
+
+/*
+ * #%L
+ * Sigmah
+ * %%
+ * Copyright (C) 2010 - 2016 URD
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/gpl-3.0.html>.
+ * #L%
+ */
+
+import java.util.Map;
+import java.util.Stack;
+
+import org.sigmah.shared.computation.dependency.Dependency;
+import org.sigmah.shared.computation.value.ComputedValue;
+
+public class Pick implements Function {
+
+	/**
+	 * {@inheritDoc}
+	 */
+	@Override
+	public String toString() {
+		return "pick";
+	}
+
+	/**
+	 * {@inheritDoc}
+	 */
+	@Override
+	public void execute(Stack<ComputedValue> stack, Map<Dependency, ComputedValue> variables) {
+		// No-op
```

Comment:
I don't get it. Can you add a Javadoc in this class explaining what is the purpose of this new "pick" function?